insights/parsers/hammer_ping.py

```diff
@@ -73,7 +73,7 @@ def parse_content(self, content):
         self.status_of_service = {}
         self.response_of_service = {}
         self.errors = []
-
+        self.raw_content = content
```

Comment:
Please add the corresponding docstring for the new attribute.

Reply:
> Please add the corresponding docstring for the new attribute.

sure, I'll add it in next commit